README.md

```diff
@@ -15,7 +15,6 @@ and ll of this are built in a beautiful and well-organized way!
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍